packages/integrations/vercel/src/serverless/adapter.ts

```diff
@@ -19,6 +19,12 @@ const PACKAGE_NAME = '@astrojs/vercel/serverless';
 export const ASTRO_LOCALS_HEADER = 'x-astro-locals';
 export const VERCEL_EDGE_MIDDLEWARE_FILE = 'vercel-edge-middleware';
 
+const SUPPORTED_NODE_VERSIONS: Record<string, { status: 'current' } | { status: 'deprecated', removal: string }> = {
```

Comment:
Would be nice to add a comment and link to https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/node-js#node.js-version so we can keep this up-to-date in the future.

Reply:
Yes, that's a great idea. Meant to do that actually.